### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

**Review by VincentLanglet — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
         "symfony/browser-kit": "^4.4 || ^5.3 || ^6.0",
         "symfony/css-selector": "^4.4 || ^5.3 || ^6.0",
         "symfony/framework-bundle": "^4.4 || ^5.3 || ^6.0",
-        "symfony/phpunit-bridge": "^5.3 || ^6.0",
```
Comment (phansys) [inline composer.json]: Is this removal on purpose?

File: src/Admin/Extension/Gedmo/TranslatableAdminExtension.php [hunk 2]
```diff
@@ -56,31 +58,91 @@ public function __construct(
 
     public function alterNewInstance(AdminInterface $admin, object $object): void
     {
+        // NEXT_MAJOR: Remove the entire "if" block.
```
Comment (phansys) [inline src/Admin/Extension/Gedmo/TranslatableAdminExtension.php:61]: I think this block should be deleted based on the `NEXT_MAJOR` comment.

File: psalm.xml [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
         </ignoreFiles>
     </projectFiles>
+    <!-- NEXT_MAJOR: Remove this issueHandlers -->
+    <issueHandlers>
+        <DeprecatedClass>
+            <errorLevel type="suppress">
+                <referencedClass name="Sonata\TranslationBundle\Model\Gedmo\TranslatableInterface"/>
+            </errorLevel>
+        </DeprecatedClass>
+    </issueHandlers>
```
Comment (franmomu) [inline psalm.xml:17]: I've added this so after merging people using `3.x` will have time to upgrade, in a few days I'll clean all the NEXT_MAJOR comments.

File: composer.json (hunk 1 above)
Comment (franmomu) [reply composer.json]: I'll put it back, I haven't checked, but I thought that package is meant to be used with just one version.

File: src/Admin/Extension/Gedmo/TranslatableAdminExtension.php (hunk 2 above)
Comment (franmomu) [reply src/Admin/Extension/Gedmo/TranslatableAdminExtension.php:61]: It was left on purpose because of https://github.com/sonata-project/SonataTranslationBundle/pull/580#discussion_r751620806

**Review by phansys — APPROVED** [review]
